### PR TITLE
Subheading onClick

### PR DESCRIPTION
**Pull request by lattai**
Updated the onClick function so that whenever a subparagraph is clicked, the respective paragraph is displayed below the flexbox. I also deleted some source code that I wasn't using. I am no longer calling the hover function, but will be using that soon.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -11,23 +11,23 @@
...
+        <br>
 
         <div class = "subParagraphs">
-        	<div onmouseover="hover()" onclick="expand()"><h2>About me</h2></div>
-        	
-        	<h2>Past Projects</h2>
-        	<h2>Links</h2>
+        	<div id= "subPar" onclick="expand('about')"><h2>About me</h2></div>
+        	<div id= "subPar" onclick="expand('projects')"><h2>Past Projects</h2></div>
+            <div id= "subPar" onclick="expand('facts')"><h2>Fun Facts</h2></div>
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/index.html:26]: formatting seems off

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -11,23 +11,23 @@
...
-        <button onclick="addRandomGreeting()">Hello</button>
-        <div id="greeting-container"></div>
+        <br>
 
         <div class = "subParagraphs">
-        	<div onmouseover="hover()" onclick="expand()"><h2>About me</h2></div>
-        	
-        	<h2>Past Projects</h2>
-        	<h2>Links</h2>
+        	<div id= "subPar" onclick="expand('about')"><h2>About me</h2></div>
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/index.html:24]: looks like you're duplicating div ids? Can they all have a unique id?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -13,26 +13,39 @@
...
+
+/**
+  * Adds filler paragraph when ANY subparagraph is clicked.
+   */
+var paragraphTitle
+function expand(info){
     const aboutMeContainer = document.getElementById('about-container');
-    aboutMeContainer.innerText = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    //Clears the paragraph text if the header for the section you're already on is clicked.
+    if (paragraphTitle == ""+info){
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/script.js:30]: Formatting seems off. Add spaces between operators.

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -4,38 +4,66 @@
...
+    padding: 0px;
+    opacity: 1;
+}
+h3 {
+    font-family: 'Permanent Marker';
+    color: white;
+    font-size: 30px;
+    text-align: center;
+    padding: 0px;
+    opacity: 1;
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/style.css:40]: You can combine the share styles here.

**Comment by lattai** [comment]
This is what it looks like right now. 
5-26 Portfolio

**Pull request merged**